oops/oops.go

```diff
@@ -6,10 +6,26 @@ import (
 	"bytes"
 	"fmt"
 	"io"
+	"os"
 	"runtime"
 	"strings"
 )
 
+var (
+	// filePrefixesToSkip is a set of prefixes of files which, when encountered when serializing
+	// an oops error, will skip appending the current frame and all subsequent frames in that stack.
+	// It is useful for truncating long call stacks to make logged oops errors more concise.
+	filePrefixesToSkip = make(map[string]struct{})
+)
+
+func init() {
+	envPrefixesToSkip := os.Getenv("OOPS_FILE_PREFIXES_TO_SKIP")
+	splitPrefixes := strings.Split(envPrefixesToSkip, ",")
+	for _, prefix := range splitPrefixes {
```

Comment:
Nit: `splitPrefixes` is only used the one time, so consider ranging over `strings.Split(envPrefixesToSkip, ",")` directly.